app/lib/features/events/sheets/edit_event_sheet.dart

```diff
@@ -46,44 +38,40 @@ class _EditEventSheetConsumerState extends ConsumerState<EditEventSheet> {
 
   // apply existing data to fields
   void _editEventData() async {
-    final calendarEvent = await ref.read(
-      calendarEventProvider(widget.calendarId!).future,
+    final calendarEvent = ref.read(
+      calendarEventProvider(widget.calendarId!),
```

Comment:
can we make this a regular await rather than a later changing-the-data-under-my-hood thing?

```suggestion
    final event = await ref.read(
      calendarEventProvider(widget.calendarId!).future,
```